app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -162,6 +164,18 @@ class BrowserActivity : DuckDuckGoActivity(), CoroutineScope {
             return
         }
 
+        var timesDefaultBrowserDialogOpened = intent.getIntExtra(TIMES_OPENED, -1)
+        if (timesDefaultBrowserDialogOpened > 1.unaryMinus()) {
```

Comment:
I'm not sure if I like that to be honest... > -1 seems way more readable to me.

Reply:
`>= 0` seems even more readable again IMO